packages/fx-core/resource/package.nls.json

```diff
@@ -374,6 +374,16 @@
   "core.common.OutlookDesktopClientName": "Outlook desktop client id",
   "core.common.OutlookWebClientName1": "Outlook web access client id 1",
   "core.common.OutlookWebClientName2": "Outlook web access client id 2",
+  "core.common.CancelledMessage": "Operation cancelled.",
+  "core.common.SpecVersionNotSupported": "OpenAPI version lower than 3.0.0 is not supported.",
+  "core.common.NoServerInformation":  "No server information found in the OpenAPI specification file.",
+  "core.common.MultipleServerInformation": "Multiple server information found in the OpenAPI specification file.",
+  "core.common.RemoteRefNotSupported": "Remote reference is not supported: %s.",
+  "core.common.MissingOperationId": "Missing operationIds: %s.",
+  "core.common.NoSupportedApi": "No supported API found in the OpenAPI specification file.",
+  "core.common.AdditionalPropertiesNotSupported": "additionalProperties is not supported, and will be ignored.",
```

Comment:
this additionalProperties is a dedicated term, in order to use the singular form, better to quote it like "additionalProperties" or 'additionalProperties'